pages/super_search_page.py

```diff
@@ -15,27 +15,29 @@ class CrashStatsSuperSearch(CrashStatsBasePage):
     _page_title = 'Search - Mozilla Crash Reports'
 
     # Search parameters section
-    _field_text_locator = (By.CSS_SELECTOR, 'fieldset[id="%s"] > div:nth-child(2) span')
+    _facet_text_locator = (By.CSS_SELECTOR, 'fieldset[id="%s"] > div:nth-child(2) span:first-child')
+    _facet_field_locator = (By.CSS_SELECTOR, 'fieldset[id="%s"] .select2-container.field')
+    _facet_field_dropdown = (By.CSS_SELECTOR, 'fieldset[id="%s"] .select2-container.field')
```

Comment:
Just testing you ;-) no, seriously, good catch. That is detritus left over from an earlier experiment.